odoo/sql_db.py

```diff
@@ -291,6 +298,15 @@ def __init__(self, pool, dbname, dsn):
         self.cache = {}
         self._now = None
 
+    def _increase_savepoint_count(self):
+        super()._increase_savepoint_count()
+        # This is not an abitrary number but linked to the size of the shared memory size that store sub trans info
+        if self.savepoint_count % 60 == 0 and odoo.registry(self.dbname).ready:
```

Comment:
Need to check if the registry is ready to avoid raising the warning during module installation due to check_indexes